pkg/services/alerting/service.go

```diff
@@ -130,6 +151,10 @@ func (s *AlertNotificationService) GetAlertNotificationsWithUidToSend(ctx contex
 func (s *AlertNotificationService) createNotifier(ctx context.Context, model *models.AlertNotification, secureSettings map[string]string) (Notifier, error) {
 	secureSettingsMap := map[string]string{}
 
+	if util.IsShortUIDTooLong(model.Uid) {
```

Comment:
Think we need to validation `NewUid` as well so maybe this validation need to happen in `UpdateAlertNotificationWithUid`?